DbTables/src/DbSet.cc

```diff
@@ -0,0 +1,83 @@
+
+#include "Offline/DbTables/inc/DbSet.hh"
+#include "cetlib_except/exception.h"
+#include <iomanip>
+#include <iostream>
+
+namespace mu2e {
+
+  //***********************************************
+
+void DbSet::add(int tid, int cid, DbIoV const& iov) { 
+  _emap[tid].emplace_back(cid, iov); 
+}
+
+  //***********************************************
+
+DbSet::EIoV DbSet::find(int tid, uint32_t run, uint32_t subrun) const {
+
+  auto iter = _emap.find(tid);
+  if (iter == _emap.end()) {
+    return EIoV();
```

Comment:
What is the behaviour if there is no match, the function returns a default constructed EIoV, the caller never checks validity and uses it anyway?

If it's required that the caller check validity, and if this class is intended only to be used inside DbService, we can leave it as is. But if it could be used outside of the DbService we should think about protection for the caller not checking.

Reply:
It is required that the code does not throw, but the code can be arranged like "bool find(...EIoV& eiov)" if you prefer.  The default EIoV has cid=-1 and IoV is null (matches no range) so should not give false results.  The existing caller does check. This pattern just reflected what I had in Engine previously..  Let me know if you prefer the other pattern.

Reply:
Thanks.  We can leave it as is.  The behaviour sounds like it's well defined.  So long as this is for internal use and the callers check, then its OK.